Application/Api/Billing/Bookkeeping/ApiBasketRepaymentAddPerson.php

```diff
@@ -0,0 +1,903 @@
+use SPHERE\Common\Frontend\Table\Structure\TableData;
+use SPHERE\Common\Frontend\Text\Repository\Bold;
+use SPHERE\Common\Frontend\Text\Repository\ToolTip;
+use SPHERE\Common\Frontend\Text\Repository\Warning as WarningText;
+use SPHERE\System\Extension\Extension;
+
+/**
+ * Class ApiBasketRepaymentAddPerson
+ * @package SPHERE\Application\Api\Billing\Bookkeeping
+ */
+class ApiBasketRepaymentAddPerson extends Extension implements IApiInterface
+{
+
+    // registered method
+    use ApiTrait;
+
+    public function exportApi($Method = '')
+    {
+        $Dispatcher = new Dispatcher(__CLASS__);
+        // AddDebtorSelection
+        $Dispatcher->registerMethod('showSearchModal');
+        $Dispatcher->registerMethod('showSearch');
+        $Dispatcher->registerMethod('addDebtorSelection');
+
+        // reload Table
+        $Dispatcher->registerMethod('getTableLayout');
+
+        // Item Price
+        $Dispatcher->registerMethod('showItemPrice');
+        $Dispatcher->registerMethod('checkItemPrice');
+        $Dispatcher->registerMethod('changeItemPrice');
+
+        // BankAccount
+        $Dispatcher->registerMethod('showEditBankAccount');
+        $Dispatcher->registerMethod('saveEditBankAccount');
+
+        // remove Entry
+        $Dispatcher->registerMethod('showDeleteBasketSelection');
+        $Dispatcher->registerMethod('deleteDebtorSelection');
+
+
+        return $Dispatcher->callMethod($Method);
+    }
+
+    /**
+     * @param string $Header
+     * @param string $Identifier
+     *
+     * @return ModalReceiver
+     */
+    public static function receiverModal($Header = '', $Identifier = '')
+    {
+
+        return (new ModalReceiver($Header, new Close()))->setIdentifier('Modal'.$Identifier);
+    }
+
+    /**
+     * @param string $Content
+     * @param string $Identifier
+     *
+     * @return BlockReceiver
+     */
+    public static function receiverSearch($Content = '', $Identifier = '')
+    {
+
+        return (new BlockReceiver($Content))->setIdentifier('Block'.$Identifier);
+    }
+
+    /**
+     * @param string $Content
+     *
+     * @return BlockReceiver
+     */
+    public static function receiverTableLayout($Content = '')
+    {
+
+        return (new BlockReceiver($Content))->setIdentifier('BasketVerificationLayout');
+    }
+
+    /**
+     * @return InlineReceiver
+     */
+    public static function receiverService()
+    {
+
+        return (new InlineReceiver())->setIdentifier('Service');
+    }
+
+    /**
+     * @param string $Content
+     * @param string $Identifier
+     *
+     * @return InlineReceiver
+     */
+    public static function receiverItemPrice($Content = '', $Identifier = '')
+    {
+
+        return (new InlineReceiver($Content))->setIdentifier('Price'.$Identifier);
+    }
+
+    /**
+     * @param string $BasketId
+     * @param string $Identifier
+     *
+     * @return Pipeline
+     */
+    public static function pipelineReloadTable($BasketId = '', $Identifier = '')
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$BasketId" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)